### DISCUSSION TITLE
Use form together with table on the same livewire component

### DISCUSSION THREAD

**Discussion by dominikgeimer** [filamentphp/filament#686] [General]
Hi,

I am using Laravel Livewire full page components (https://laravel-livewire.com/docs/2.x/rendering-components#page-components) and I want to use a form and a table on that same component, but there is a collision error. Is it not possible to use both on the same component?

```
Symfony\Component\ErrorHandler\Error\FatalError
Trait method Filament\Tables\Concerns\InteractsWithTable::getForms has not been applied as App\Http\Livewire\Team::getForms, because of collision with Filament\Forms\Concerns\InteractsWithForms::getForms
```

Thanks! 
Dominik

**Comment by danharrin** [comment]
The tables package already implements the forms interface and trait, so you should be able to use the package without adding those again to your component.

However, you'll need to override `getForms()` to inherit the forms from `InteractsWithTable` as well as your own.

**Reply by dominikgeimer** [reply]
Many thanks for your response. I'll try to see if I can do it.

**Reply by danharrin** [reply]
@DominikGeimer I've just written some documentation about this :)

You'll need to be on v2.4.12 though.

https://filamentadmin.com/docs/2.x/tables/building-tables#using-the-form-builder

**Reply by dominikgeimer** [reply]
@danharrin You are so awesome! Just updated to v2.4.12 and it works wonderfully!

**Reply by danharrin** [reply]
I'm glad! Happy building 👷‍♂️
